src/librustc_data_structures/lib.rs

```diff
@@ -58,6 +60,26 @@ extern crate rustc_cratesio_shim;
 
 pub use rustc_serialize::hex::ToHex;
 
+#[macro_export]
+macro_rules! likely {
```

Comment:
This macro isn't used.

Reply:
It gets used in https://github.com/rust-lang/rust/pull/56614. Might as well add them both

Reply:
Didn't look at that PR :)